External/FEXCore/Source/Interface/Context/Context.cpp

```diff
@@ -193,6 +193,10 @@ namespace FEXCore::Context {
     return CTX->UnloadAOTIRCacheEntry(Entry);
   }
 
+  uintptr_t AddCustomIREntrypoint(FEXCore::Context::Context *CTX, uintptr_t Entrypoint, std::function<void(uintptr_t Entrypoint, FEXCore::IR::IREmitter *)> Handler, const void *Creator, uintptr_t Data) {
```

Comment:
Should return `std::optional<uintptr_t>` so LoadIR can tell if custom IR had been registered here before or not.